source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -230,21 +236,21 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 					telemetryBrowser.capture("EDITOR cloned repository", {
 						owner: args.routeParams.owner,
 						repository: args.routeParams.repository,
-						isPrivate: response.data.private,
+						type: response.data.private ? "isPrivate" : "isPublic",
```

Comment:
I would only use "private" or "public" without the `is` prefix. The `is` prefix is used for boolean values, making it easier to identify that a variable named `isPrivate` is a boolean that "is" either true or false.